index.js

```diff
@@ -9,20 +9,29 @@ const mix = (c1, c2, w='') => {
   return mixed.alpha() < 1 ? mixed.rgbaString() : mixed.hexString();
 };
 
-const transformColor = (string, source) => {
-  if (string.indexOf('mix(') === -1) {
-    return string;
+const shade = (c, w) => mix('#000', c, w);
+
+const tint = (c, w) => mix('#fff', c, w);
+
+const getTransform = (string) => {
+  if (string.indexOf('shade(') !== -1) {
+    return shade;
+  } else if (string.indexOf('tint(') !== -1) {
+    return tint;
+  } else {
+    return mix;
   }
+};
 
+const transformColor = (string, source) => {
+  const transform = getTransform(string);
   const value = balanced('(', ')', string).body;
 
-  if (!value) { throw new Error(`Missing closing parentheses in "${string}"`, source); }
```

Comment:
No, it can’t. Missed parentheses causes CSS parse error; in this case `transformColor` is not invoked.

Empty string (e.g. `mix()`) results in color parse error:
```
Error: <css input>:1:5: Unable to parse color from string ""
```